src/builder.rs

```diff
@@ -1326,7 +1326,10 @@ fn setup_logger(
 			Logger::new_fs_writer(log_file_path, max_log_level)
 				.map_err(|_| BuildError::LoggerSetupFailed)?
 		},
-		Some(LogWriterConfig::Log(log_level)) => Logger::new_log_facade(*log_level),
+		Some(LogWriterConfig::Log { max_log_level }) => {
+			let max_log_level = max_log_level.unwrap_or_else(|| DEFAULT_LOG_LEVEL);
+			Logger::new_log_facade(max_log_level)
```

Comment:
One thing that I noticed about the log facade is that you can set log level GOSSIP, but that isn't visible anymore after this PR. The standard log only knows TRACE of course. Not a problem I think?

And perhaps some explanation on how the `RUST_LOG` env var interacts with this log level would be helpful too. It wasn't very clear to me how that works.